ivy/functional/frontends/tensorflow/nn.py

```diff
@@ -123,3 +123,7 @@ def batch_normalization(x, mean, variance, offset, scale, variance_epsilon, name
     return x * ivy.astype(inv, x.dtype, copy=False) + ivy.astype(
         offset - mean * inv if offset is not None else -mean * inv, x.dtype
     )
+
+
+def convolution(input, filters, strides=None, padding="VALID", data_format="NDHWC", dilations=None, name=None):
+    return ivy.convolution(input, filters, strides, padding, data_format=data_format, dilations=dilations)
```

Comment:
I don't think there's a `convolution` function in Ivy's functional API, so either you'll have to implement the `tensorflow` `convolution` frontend as a composition of existing Ivy functions or extend Ivy's functional API by adding `ivy.convolution` in a separate PR referring to the [docs](https://lets-unify.ai/ivy/contributing/4_open_tasks.html#ivy-api-extensions)

Reply:
Okay, I want to implement the tensorflow convolution frontend as a composition of existing Ivy functions, How can I do that?